index.html

```diff
@@ -5,20 +5,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Powerhouse Connect</title>
         <base href="/">
+        %VITE_IMPORT_REACT_SCRIPT%
```

Comment:
using this instead of vite-html-plugin (it doesn't work for some weird reason, we need to investigate what's wrong with this dep)